packages/cli/src/commands/exec.js

```diff
@@ -41,6 +41,12 @@ export const builder = (yargs) => {
       default: true,
       description: 'Generate the Prisma client',
     })
+    .option('list', {
+      alias: 'ls',
```

Comment:
To me it would be more natural to have `--list` and `-l` (instead of `--list` and `--ls`)

Reply:
You got it!